services/web/server/src/simcore_service_webserver/projects/projects_exceptions.py

```diff
@@ -35,10 +36,19 @@ def __init__(self, project_uuid):
         self.project_uuid = project_uuid
 
 
+class ProjectDeleteError(ProjectsException):
+    def __init__(self, project_uuid, reason="Unknown"):

```

Comment:
I think this unknown reason for an exception si not that helpful. Especially when trying to debug an issue. If you want to have it why not always force it?

Reply:
Good point. I will change that